test/over_react/component/error_boundary_test.dart

```diff
@@ -145,6 +167,446 @@ void main() {
           expect(jacket.getNode(), hasNodeName('H4'));
           expect(jacket.getNode().text, 'Something super not awesome just happened.');
         });
+
+        group('and then switches back to rendering the child', () {
+          setUp(() {
+            jacket = mount((ErrorBoundary()
+              ..fallbackUIRenderer = (_, __) => (Dom.div()..addTestId('fallbackNode'))('Something went wrong')
+            )(dummyChild));
+            final component = jacket.getDartInstance();
+            component.setState(component.newState()
+              ..hasError = true
+              ..showFallbackUIOnError = true
+            );
+            expect(getByTestId(jacket.getInstance(), 'dummyChild'), isNull);
+            expect(getByTestId(jacket.getInstance(), 'fallbackNode'), isNotNull);
+          });
+
+          test('when reset() is called', () {
+            jacket.getDartInstance().reset();
+            expect(jacket.getDartInstance().state.hasError, isFalse);
+            expect(jacket.getDartInstance().state.showFallbackUIOnError, isTrue);
+            expect(getByTestId(jacket.getInstance(), 'dummyChild'), isNotNull);
+            expect(getByTestId(jacket.getInstance(), 'fallbackNode'), isNull);
+          });
+
+          test('when a new child is passed in', () {
+            final newDummyChild = (Dom.div()..addTestId('newDummyChild'))('hi there');
+            jacket.rerender((ErrorBoundary()
+              ..fallbackUIRenderer = (_, __) => (Dom.div()..addTestId('fallbackNode'))('Something went wrong')
+            )(newDummyChild));
+            expect(jacket.getDartInstance().state.hasError, isFalse);
+            expect(jacket.getDartInstance().state.showFallbackUIOnError, isTrue);
+            expect(getByTestId(jacket.getInstance(), 'newDummyChild'), isNotNull);
+            expect(getByTestId(jacket.getInstance(), 'fallbackNode'), isNull);
+          });
+        });
+      });
+    });
+
+    group('gracefully handles errors in its tree when `props.fallbackUIRenderer` is not set', () {
+      List<Map<String, List>> calls;
+      DivElement mountNode;
+      var flawedRenderedInstance;
+      var nestedFlawedRenderedInstance;
+      const identicalErrorFrequencyToleranceInMs = 100;
```

Comment:
This value should probably be a little bit larger to reduce flakiness on slow machines. Maybe 500ms? Or, you could alternatively add a `retry` to the group.

Also #nit making this a `Duration` could make this easier to work with and reduce the chance of messing up units

Reply:
Addressed in https://github.com/Workiva/over_react/pull/350/commits/a9e864eeef2abd045c977aa1377569d7fcb3d399